assets/sass/06-components/404.scss

```diff
@@ -1,15 +1,4 @@
-.error-404 {
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	height: 60vh;
-	flex-direction: column;
-}
-
-.error404 .page-header {
-	border-bottom: 3px solid transparent;
-}
-
-.error404 p {
+.error404 main p {
 	font-size: var(--global--font-size-lg);
+	margin-bottom: calc(var(--global--font-size-base) * 2.5);
```

Comment:
Is there a reason why the vertical spacing variable is not used here?
(https://github.com/WordPress/twentytwentyone/blob/trunk/assets/sass/01-settings/global.scss#L92)

Reply:
> Is there a reason why the vertical spacing variable is not used here?
> (https://github.com/WordPress/twentytwentyone/blob/trunk/assets/sass/01-settings/global.scss#L92)

This happened accidentally and I just replaced `--global--font-size-base` with `--global--spacing-vertical`.